### ISSUE TITLE
PyPi Release?

### ISSUE THREAD

**Issue by vincecarney** [zostera/django-bootstrap4#51]
I'd like to incorporate fix for #39. Can we get a new PyPi release soon?

https://pypi.python.org/pypi/django-bootstrap4

**Comment by FlipperPA** [comment]
Just in case you (or anyone else) doesn't know, but you can `pip install` from a git URL, even at a specific commit. This also works for requirements files.

To install the latest master:

```bash
pip install git+https://github.com/zostera/django-bootstrap4.git
```
To install at this commit:

```bash
pip install git+https://github.com/zostera/django-bootstrap4.git@79a0f68864dc5d971897d5cdd7ce676c8dd3f7c6
```

For requirements file, just drop the `pip install`. That's what I use until the merge I need is published to PyPI.

**Comment by jieter** [comment]
released [0.0.5](https://pypi.python.org/pypi/django-bootstrap4/0.0.5)
